keystatic-implementation-prompt.md

```
@@ -0,0 +1,126 @@
+# Keystatic Integration — Claude Code Implementation Prompt
+
+Use this prompt in Claude Code from the project root (`gwilym.ca/`).
+
+---
+
+## Prompt
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)